elements/lisk-chain/src/constants.ts

```diff
@@ -33,7 +33,7 @@ export const TAG_BLOCK_HEADER = utils.createMessageTag('BH');
 export const TAG_TRANSACTION = utils.createMessageTag('TX');
 
 // TODO: Actual size TBD
-export const MAX_ASSET_DATA_SIZE_BYTES = 64;
+export const MAX_ASSET_DATA_SIZE_BYTES = 18;
```

Comment:
I am not sure about this, I assume this is ok with @shuse2

Reply:
I think it should be ok. (we only use it for seed reveal as of now)

Reply:
I was just following the specs. [The LIP-0063 says the value should be 18](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#constants-1).

Reply:
Ok, did not catch that it was specified in LIP 63..